tests/baselines/reference/complexRecursiveCollections.errors.txt

```diff
@@ -339,6 +347,15 @@ immutable.ts(391,22): error TS2430: Interface 'Set<T>' incorrectly extends inter
         export function Set<T>(): Seq.Set<T>;
         export function Set<T>(collection: Iterable<T>): Seq.Set<T>;
         export interface Set<T> extends Seq<never, T>, Collection.Set<T> {
+                         ~~~
+!!! error TS2430: Interface 'Set<T>' incorrectly extends interface 'Seq<never, T>'.
+!!! error TS2430:   The types of 'map(...).filter(...).concat(...).toMap().toSeq().toOrderedMap().keySeq().toMap().mapKeys(...).toSeq().toOrderedMap().toSet().union' are incompatible between these types.

```

Comment:
😵

Reply:
We go hard at finding the incompatibility, don't we. 😆

Guess there's an `any` somewhere in this type definition that should be a `never` (or vice-versa, depending on intent).